src/govuk/components/accordion/template.njk

```diff
@@ -1,8 +1,15 @@
 {% set id = params.id %}
 {% set headingLevel = params.headingLevel if params.headingLevel else 2 %}
 
-<div class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}" data-module="govuk-accordion" id="{{ id }}"
-{%- for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+<div
+  class="govuk-accordion {%- if params.classes %} {{ params.classes }}{% endif -%}"
+  data-module="govuk-accordion"
+  id="{{ id }}"
+  {%- if params.hideAllSectionsHtml %} data-i18n.hide-all-sections="{{ params.hideAllSectionsHtml | escape }}"{% endif %}
+  {%- if params.hideSectionHtml %} data-i18n.hide-section="{{ params.hideSectionHtml | escape }}"{% endif %}
+  {%- if params.showAllSectionsHtml %} data-i18n.show-all-sections="{{ params.showAllSectionsHtml | escape }}"{% endif %}
+  {%- if params.showSectionHtml %} data-i18n.show-section="{{ params.showSectionHtml | escape }}"{% endif %}
```

Comment:
Can we add tests for these to template.test.js?

Reply:
Can I check what the template tests need to cover? Making sure the values are escaped? 
I've already added tests to ensure that the UI text matches the data-* attributes to integration tests.

Reply:
I've added tests to check for the existence and values of the data-* attributes. I couldn't find a way to test that their values were being properly escaped by the Nunjucks filter, however, as the HTML seems to have already been 'transformed' back into regular HTML by the time the Jest is looking at it.

Reply:
Aah of course, that makes sense. I guess if they weren't being escaped correctly the attributes would be malformed and the values would be incorrect?

Reply:
Yep, that's the best we can hope for, I think.